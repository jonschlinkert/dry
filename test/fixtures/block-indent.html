{% extends "block-indent" %}

  {% block "header" %}
  header: child
  {% endblock %}

{% block "middle" %}
middle: child
{% endblock %}

  {% block "footer" %}
  footer: child
  {% endblock %}