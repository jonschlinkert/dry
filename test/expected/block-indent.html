unique: parent

  
  header: child
  
    
middle: child

      
  footer: child
  
  
  foot: parent
  
